paint/preprocessing/calibration_stac.py

```diff
@@ -139,7 +143,7 @@ def make_calibration_item(
         "stac_extensions": [mappings.VIEW_EXTENSION, mappings.PROCESSING_EXTENSION],
         "id": f"{image}",
         "type": "Feature",
-        "title": f"Calibration data from heliostat {heliostat_data[mappings.HELIOSTAT_ID]} for image {image}",
+        "title": f"Calibration preprocessing from heliostat {heliostat_data[mappings.HELIOSTAT_ID]} for image {image}",
```

Comment:
Not sure about this, but I suspect this should also be "data"? Or does this refer to the preprocessed data specifically here?